apis/Google.Area120.Tables.V1Alpha1/docs/history.md

```diff
@@ -0,0 +1,6 @@
+# Version history
+
+# Version 1.0.0-alpha01, released 2020-09-22
+
+- First alpha release.
```

Comment:
I'd probably not bother with a "-" here. It's not an  entry in a bulleted list of changes; it's explaining why there aren't any changes. Definitely not a big deal though (and I may well have done something different at other times myself).

Reply:
Done. Will merge on green.